lorahw/gateway.c

```diff
@@ -140,8 +139,7 @@ int setUpGateway(int bus, int region) {
     // maps the rf power levels to sx1302 chip specific power control registers.
     uint8_t pwr_idx [16] = {15,16,17,19,20,22,1,2,3,4,5,6,7,9,11,14};
 
-    // sx1302 supports 16 power levels
```

Comment:
Keep comment, it's useful context that this c code is largely dealing with a sx1302 concentrator